client/state/sites/selectors.js

```diff
@@ -365,9 +365,9 @@ export function isRequestingSites( state ) {
  * Returns true if a network request is in progress to fetch the specified, or
  * false otherwise.
  *
- * @param  {Object}  state  Global state tree
- * @param  {Number}  siteId Site ID
- * @return {Boolean}        Whether request is in progress
+ * @param  {Object}           state  Global state tree
+ * @param  {(Number|String)}  siteId Site ID or slug
+ * @return {Boolean}          Whether request is in progress
  */
 export function isRequestingSite( state, siteId ) {
```

Comment:
It probably makes sense to add a test for the string case, too.

Reply:
As discussed via DM:

We currently have 3 unit tests covering `isRequestingSite` (using a numeric `siteId`):

https://github.com/Automattic/wp-calypso/blob/9243259bb4231c615c42ee9db009e2bfdd4ff088/client/state/sites/test/selectors.js#L1109-L1152

I don't think it makes sense to add only _one_ test for `siteSlug` strings (e.g. only for the "request in progress" case), since that might lead to the wrong assumption that the selector is implemented differently for strings (which it isn't). If anything, we'd have to duplicate all three tests for strings. However, that seems really redundant since `isRequestingSite` is really completely type-agnostic; and I think the place to document this is the JSDoc, which we're already doing 😄